pkg/cmd/server/api/types.go

```diff
@@ -196,6 +196,14 @@ type MasterConfig struct {
 	ControllerLeaseTTL int
 	// TODO: the next field added to controllers must be added to a new controllers struct
 
+	// KubeAdmissionControlPluginsOverride is a list of admission control plugins that will be installed
+	// on the Kubernetes master. Order is significant. If empty, a default list of plugins is used.
+	KubeAdmissionControlPluginsOverride []string
```

Comment:
It helps to understand that if this is left blank a default set of plugins will be used